ding/worker/collector/base_serial_evaluator.py

```diff
@@ -89,11 +89,12 @@ def __init__(self, env_num: int, n_episode: int) -> None:
         assert n_episode >= env_num, "n_episode < env_num, please decrease the number of eval env"
         self._env_num = env_num
         self._n_episode = n_episode
-        each_env_episode = [n_episode // env_num for _ in range(env_num)]
+        self._each_env_episode = [n_episode // env_num for _ in range(env_num)]
         for i in range(n_episode % env_num):
-            each_env_episode[i] += 1
-        self._reward = {env_id: deque(maxlen=maxlen) for env_id, maxlen in enumerate(each_env_episode)}
-        self._info = {env_id: deque(maxlen=maxlen) for env_id, maxlen in enumerate(each_env_episode)}
+            self._each_env_episode[i] += 1
+        self._video = {env_id: [] for env_id, maxlen in enumerate(self._each_env_episode)}
```

Comment:
use deque here so that you don't need to clip by hand in line 143, and keep the correct reward relationship

Reply:
changed